docs/installation.adoc

```diff
@@ -1,14 +1,36 @@
 == Installation
 
+=== Don't make me think
+
+Ok, just execute following snippet and you are all set:
+
+[[installing-release]]
+`curl -sSL https://git.io/v5jy6 | bash` copyToClipboard:installing-release[]
+
+You can also add these flags:
+
+  * `-l|--latest` latest published artifact in Maven Central
+  * `-r|--release` latest release (that's the default)
```

Comment:
what is the difference between: ` latest published artifact in Maven Central` and  `latest release`?
For me, they seem to be same

Reply:
you could have betas etc

Reply:
See here http://central.maven.org/maven2/org/arquillian/smart/testing/smart-testing-parent/maven-metadata.xml

Reply:
But maybe this is not really needed - but I was thinking we can at some point have betas or even make this link configurable so it can point to local repo or on-premise nexus - just a flexibility.

Reply:
But yeah, looking at other projects in practice there is no distinction http://central.maven.org/maven2/io/netty/netty-parent/maven-metadata.xml.

Reply:
Shall we drop one? `release` or `latest`?

Reply:
Nice monologue :-D
betas and alphas are releases in maven central as well, so that wouldn't be clear at all.
It would make sense in case you would like to take the latest commit (state) from the GH project, but this is not necessary I think.
So, I would drop the `release` and use just `latest`

Reply:
it seems to only be working from plugins